apps/src/templates/teacherDashboard/OwnedSections.jsx

```diff
@@ -61,6 +62,11 @@ class OwnedSections extends React.Component {
   constructor(props) {
     super(props);
     this.onEditSection = this.onEditSection.bind(this);
+    if (experiments.isEnabled(experiments.TEACHER_DASHBOARD_SECTION_BUTTONS)) {
+      recordImpression('OwnedSectionsTableTeacherHeaderWithButtons');
+    } else {
```

Comment:
nit, but I would use snake case here since that seems to be our convention for firehose

Reply:
done! good catch